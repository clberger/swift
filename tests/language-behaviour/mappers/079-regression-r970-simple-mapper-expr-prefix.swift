type messagefile;

(messagefile t) write() { 
    app {
        echo @filename(t) stdout=@filename(t);
    }
}

string fn = "079-regression-r970-simple-mapper-expr-prefix";

messagefile outfile <simple_mapper;prefix=fn,suffix=".out">;

outfile = write();

