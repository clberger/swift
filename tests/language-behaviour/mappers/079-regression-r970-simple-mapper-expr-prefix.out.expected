079-regression-r970-simple-mapper-expr-prefix.out
